src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -257,5 +257,16 @@ public static string GetFullPath(string path)
 
             return Path.GetFullPath(path);
         }
+
+        public static string GetCurrentDirectoryWithDirSeparator()
```

Comment:
We already have this: https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.Cli.Utils/PathUtility.cs#L33.